README.md

```diff
@@ -155,7 +154,16 @@ Than  you can run tests:
 
 `source ./set-env.mixed.sh && docker-compose run --rm dev vendor/bin/phpunit --testsuite backend-mixed'`
 
+## Running test from PHPStorm
 
+The whole test suite is quite big and it can take few hours. So it is a good idea to run just a testcase which you are interested in from [PHPStorm](https://blog.jetbrains.com/phpstorm/2017/07/new-docker-compose-support-in-phpstorm-2017-2/).
```

Comment:
Moc nechápu, jakou to má souvislost - link a to o čem mluvíš. Kdyžtak to měl být link na PHPStorm dokumentaci, ne?